cpp/src/plasma/eviction_policy.cc

```diff
@@ -152,6 +156,13 @@ void EvictionPolicy::RemoveObject(const ObjectID& object_id) {
   cache_.Remove(object_id);
 }
 
+void EvictionPolicy::RefreshObjects(const std::vector<ObjectID>& object_ids) {
+  for (const auto& object_id : object_ids) {
+    int64_t size = cache_.Remove(object_id);
+    cache_.Add(object_id, size);
```

Comment:
Shouldn't we be checking if size != -1 here or does Add check that explicitly?

Reply:
Good point, I added the check!